keras_cv/layers/object_detection3d/center_net_label_encoder.py

```diff
@@ -0,0 +1,447 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+from typing import Sequence
+from typing import Tuple
+from typing import Union
+
+import numpy as np
+import tensorflow as tf
+
+from keras_cv.layers.object_detection3d import voxel_utils
+
+# Infinite voxel size.
+INF_VOXEL_SIZE = 100
+
+
+def _meshgrid(
+    max_radius_in_voxels: Sequence[int], voxel_size: Sequence[float]
+) -> np.ndarray:
+    """Computes the mesh grid given number of points in each dimension.
+
+    NOTE: this is a pure numpy function.
```

Comment:
Can this work as is with tf.numpy?

Reply:
We probably don't want to rely on experimental features?

Reply:
https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/stable_diffusion/text_encoder.py#L17

In any case they are generally python "wrapped" on TF ops so it is just a quick proxy to check if it could be expressed in TF ops.